domino-ui/src/main/java/org/dominokit/domino/ui/forms/BasicFormElement.java

```diff
@@ -198,113 +199,113 @@ public T clearInvalid() {
         }
         return (T) this;
     }
-
+    
     private void removeErrors() {
         errorLabels.clear();
         getErrorsContainer().clearElement();
         removeCss("error");
     }
-
+    
     public List<HTMLElement> getErrorLabels() {
         return errorLabels;
     }
-
+    
     @Override
     public T setRequired(boolean required) {
+        if(nonNull(requiredIndicator) && nonNull(getLabelElement()) && getLabelElement().contains(requiredIndicator)){
```

Comment:
`getLabelElement().contains(requiredIndicator)` i thought this will only work when the element is attached?

Reply:
Yes, you are right. I had a 'This is not a child node'-isssue. So I added it. Dig deeper into it.